livro/inc/gutenberg-dependency-check.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+add_action( 'admin_notices', 'show_admin_messages' );
+
+function show_admin_messages() {
+	$metadata = file_get_contents( get_stylesheet_directory() . '/style.css' );
+	preg_match( '/(?<=Requires Gutenberg:).+/', $metadata, $match );
+	if ( 0 === sizeof( $match ) ) {
+		return;
+	}
+	$version = $match[0];
+	if( defined( 'IS_GUTENBERG_PLUGIN' ) ) {
```

Comment:
I _think_ we can simplify to something like:

```suggestion
	if( defined( 'IS_GUTENBERG_PLUGIN' ) && defined( 'GUTENBERG_VERSION' ) ) {
		if ( version_compare( GUTENBERG_VERSION, trim( $version ) ) >= 0 ) {
			return;
		}
	}
```

Reply:
Or even just `GUTENBERG_VERSION`?

Reply:
Shoot, I merged that suggestion too quickly.  GUTENBERG_VERSION doesn't seem to be defined when in development mode...